### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by will-moore**
Updates changelog with same items in 3.2.0 release blog post https://github.com/ome/omero-figure/pull/261

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+3.2.0 (January 2018)
+--------------------
+
+ - Cropping of the page to fit around image panels (PR #252)
+ - Creation of new labels from Tags on the image (PR #254)
+ - Allow renaming of saved figures (PR #251)
+ - Fix export of multi-page PDF info section with Reportlab 3.4.0 (PR #260)
```
Comment (jburel) [inline CHANGELOG.md]: reportlab

Comment (jburel) [reply CHANGELOG.md]: @will-moore could you fix the name

Comment (will-moore) [reply CHANGELOG.md]: Fixed to ReportLab

**Comment by jburel** [comment]
Thanks

**Pull request merged**